README.md

```diff
@@ -116,6 +116,23 @@ if (window.callPhantom) {
 }
 ```
 
+## Changing `viewportSize`
+
+`mocha-phantomjs-core` now also supports changing of viewportSize (the simulated `window` size for the headless browser) - while running tests.
+
+```javascript
+if (window.callPhantom) {
+  window.callPhantom({
+    viewportSize : {
+      x : 100,
+      y : 100
```

Comment:
Using `x` and `y` here when `width` and `height` is used on the API level is confusing. Change it to `width` and `height` and mention they are optional.